hedera-mirror-rest/__tests__/model/signatureType.test.js

```diff
@@ -0,0 +1,33 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+'use strict';
+
+const {SignatureType} = require('../../model');
+
+describe('getName', () => {
+  test('Valid', () => {
+    expect(SignatureType.getName(3)).toEqual('ED25519');
+  });
+
+  test('Unknown', () => {
+    expect(SignatureType.getName(9999999)).toEqual('UNKNOWN');
```

Comment:
Add below value and maybe non integer cases e.g. 
```suggestion
    expect(SignatureType.getName(1)).toEqual('UNKNOWN');
    expect(SignatureType.getName(9999999)).toEqual('UNKNOWN');
    expect(SignatureType.getName('sig')).toEqual('UNKNOWN');
```

Reply:
Added those and more